dev-docs/modules/iabCatalogTranslation.md

```diff
@@ -0,0 +1,74 @@
+---
+layout: page_v2
+page_type: module
+title: Module - IAB Category Translation
+description: Converts between ad agency brand categories and IAB brand categories.
+module_code : CategoryTranslation
+display_name : CategoryTranslation
+enable_download : true
+sidebarType : 1
+---
+
+# IAB Category Translation
+
+{:.no_toc}
+
+This module supports the mapping between ad agency brand categories and IAB subcategories and conversion of IAB subcategories to Freewheel industry group identifiers. 
```

Comment:
This module converts the IAB sub category to Freewheel industry group identifiers.

Reply:
Revised doc.